source/help/settings/theme-colors.rst

```diff
@@ -0,0 +1,138 @@
+.. _theme-colors:
+
+Theme Colors
+============
+
+The colors of the Mattermost user interface are customizable in **Account Settings > Display > Theme**. You can import your theme colors from Slack, customize the colors yourself, or choose from four standard themes designed by the Mattermost team.
+
+In Enterprise Edition, if you belong to multiple teams, you can choose to apply any changes across teams by selecting "Apply New Theme to All Teams" before saving. If this is not selected, any changes are only applied to the current team.
```

Comment:
I'm not sure if we should change this where you have the option to apply the theme on a per-team basis.

Because in TE, `you can choose to apply any changes across teams by selecting` actually works, which is the default behaviour.